src/test_utils/fixtures.ts

```diff
@@ -0,0 +1,118 @@
+import { Server } from 'node:http';
+import { fork } from 'node:child_process';
+import { AddressInfo } from 'node:net';
+import { join } from 'node:path';
+import { expect, test as originalTest } from 'vitest';
+import strip from 'strip-ansi';
+import emocks from 'emocks';
+import express from 'express';
+
+import { log } from '../log';
+
+type Fixtures = {
+  runMockServer: (mockDir: string) => Promise<Server>;
+  testCliCommand: (
+    args: string[],
+    expected?: {
+      snapshot?: true;
+      stderr?: string | ReturnType<typeof expect.stringMatching>;
+      code?: number;
+    },
```

Comment:
No `stdout`? Is that intentional? Think its duplicating the snapshot check 🤔

Reply:
Yes, it is.
Let me quote myself:

> removed unnecessary stdout check (we're already getting snapshots) / snapshot: true (we set it to true in every case) options

Reply:
the idea was, that not all tests would work well with snapshots, but it wasn't proven :)